parmed/openmm/parameters.py

```diff
@@ -101,10 +107,20 @@ def from_parameterset(cls, params, copy=False):
         new_params.parametersets = params.parametersets
         new_params._combining_rule = params.combining_rule
         new_params.residues = params.residues
+        # Assign Lennard-Jones atom classes
+        ljtypes = dict()
+        counter = 0
+        for name, atom_type in iteritems(new_params.atom_types):
+            key = (atom_type.rmin, atom_type.epsilon, atom_type.rmin_14,
+                    atom_type.epsilon_14)
+            if key not in ljtypes:
+                ljtypes[key] = counter
+                counter += 1
+            atom_type.number = ljtypes[key]
 
         return new_params
 
-    def write(self, dest, provenance=None):
+    def write(self, dest, original_file='', reference=''):
```

Comment:
Maybe it should take either a `str` (pre-formatted XML block) or a `dict` (key/value pairs)?